sufia-models/app/models/sufia/works/curation_concern/with_editors.rb

```diff
@@ -0,0 +1,45 @@
+module Sufia::Works
```

Comment:
Do we need this class?  hydra-access-controls provides the edit_user & edit_group